edx/edx-enterprise
# Updated EnterpriseCustomerReportingConfiguration model with PGP key

**Description:** Updated EnterpriseCustomerReportingConfiguration model with PGP key

**JIRA:** https://openedx.atlassian.net/browse/ENT-1273

**Testing instructions:**
1. Open http://business.sandbox.edx.org/admin/enterprise/enterprisecustomerreportingconfiguration/add/
2. Verify PGP key field
3. Add and save PGP key
4. Reopen the record to verify that the key was saved
